src/actions/index.js

```diff
@@ -0,0 +1,9 @@
+import { CREATE_BOOK, REMOVE_BOOK } from '../actionTypes';
+
+export const createBook = book => ({ // eslint-disable-line
```

Comment:
**Inline Comment:**
With the ESLint inline comment, we are indirectly changing the rules by disabling these rules, let's fix the codebase instead.

If you are using VSCode, you can delete the inline comment and hover over the error(configure eslint checks locally). And you will see your options. Click on the more info button that will take you to a GitHub repo where it is explained why you are getting this error. I find this feature very educational. Please, try to learn and implement the necessary rules so you don't have to keep this disabling line on top of each file. I believe you can do this! 💪😇

Drop a comment if you're unable to fix it.